pkg/exporter/sumologicexporter/exporter.go

```diff
@@ -55,6 +55,13 @@ const telegrafTranslationDeprecationBanner = `
 ***********************************************************************************************************************************************************
 `
 
+const sourceTemplatesDeprecationBanner = `
+***********************************************************************************************************************************************************
+***    Adding source headers is deprecated and is going to be dropped soon. Please see the migration document:                              			***
+***    https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/Upgrading.md#sumologic-exporter-drop-support-for-source-headers.    		***
```

Comment:
Some spaces to fix :)

Reply:
I guess that VS Code tricks me, it looks fine in the IDE 😦